### PR TITLE
chore(ci): update release workflow

### PR DESCRIPTION
**Pull request by kaminek**
[follow official doc](https://learn.hashicorp.com/tutorials/terraform/provider-release-publish?utm_offer=ARTICLE_PAGE)

Signed-off-by: Amine Kherbouche <kaminek92@gmail.com>

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        with:
+          go-version: 1.16
+      -
+        name: Import GPG key
+        id: import_gpg
+        uses: hashicorp/ghaction-import-gpg@v2.1.0
+        env:
+          # These secrets will need to be configured for the repository:
+          GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
+          PASSPHRASE: ${{ secrets.PASSPHRASE }}
```
Comment (Darep) [inline .github/workflows/release.yml:39]: Was the signing key & passphrase added to GitHub secrets already? Is they key a self-generated one?

Comment (kaminek) [reply .github/workflows/release.yml:39]: not yet I'll ask the ones that used to release manually

**Review by moitias — APPROVED** [review]
LGTM :+1:

File: .github/workflows/release.yml (hunk 1 above)
Comment (Darep) [reply .github/workflows/release.yml:39]: Ah ok, so we'll use the same keys we used for releasing before :) I wonder where those are...

**Pull request merged**